src/Modal/_ModalDialog.scss

```diff
@@ -310,7 +311,6 @@
   }
 
   .pgn__modal-title {
-    font-size: $h4-font-size;
```

Comment:
[inform] Per Figma, it appears all modal dialog types (including `AlertModal` should have title font size of $h3-font-size`.